web/server/routers/post.ts

```diff
@@ -66,13 +65,22 @@ const postSchema = z.object({
   ),
 });
 
+const fosterTypeEmails: Record<FosterType, string> = {
+  [FosterType.FosterMove]: "foster@angelsrescue.org",
+  [FosterType.Return]: "returns@angelsrescue.org, foster@angelsrescue.org",
+  [FosterType.Temporary]: "tempfoster@angelsrescue.org",
+  [FosterType.Boarding]: "boardingadmin@angelsrescue.org",
+  [FosterType.Shelter]: "fosteroffer@angelsrescue.org",
+  [FosterType.OwnerSurrender]: "fosteroffer@angelsrescue.org",
+};
+
```

Comment:
Add `as const` to the end of this line (const assertion). We want to let TypeScript know that we don't intent to change `fosterTypeEmails`, so the values will remain as defined here.